com.unity.ml-agents/Editor/RayPerceptionSensorComponentBaseEditor.cs

```diff
@@ -17,12 +17,12 @@ protected void OnRayPerceptionInspectorGUI(bool is3d)
             EditorGUI.BeginChangeCheck();
             EditorGUI.indentLevel++;
 
-            EditorGUILayout.PropertyField(so.FindProperty("m_SensorName"), true);
-
-            // Because the number of rays and the tags affect the observation shape,
-            // they are not editable during play mode.
+            // Don't allow certain fields to be modified during play mode.
+            // * SensorName affects the ordering of the Agent's observations
+            // * The number of tags and rays affects the size of the observations.
             EditorGUI.BeginDisabledGroup(Application.isPlaying);
             {
+                EditorGUILayout.PropertyField(so.FindProperty("m_SensorName"), true);
```

Comment:
I think it would make more sense to store "m_SensorName" and such in private const in case they are used somewhere else.

Reply:
Sorry, not quite sure what you mean. The user needs to assign a unique name to each sensor so that we can sort them deterministically, so the name here can't be const.

Reply:
i think he means the string, make it a variable.

Reply:
Here or in the component? We currently only use them in one place, and it's kinda redundant because of reflection.

We could do something like this:
https://stackoverflow.com/a/33059272/224264
although to use `nameof` we'd need to make the fields internal.

Reply:
If we only use it here, I think it is fine.